authenticator/server/authenticator.go

```diff
@@ -254,7 +171,7 @@ func (a *Authenticator) GetPGMD5Hash(ctx context.Context, req *pb.PGMD5HashReque
 	return &pb.PGMD5Response{Hash: hash}, nil
 }
 
-func (a *Authenticator) GetPGSHA256Hash(ctx context.Context, req *pb.PGSHA256HashRequest) (*pb.PGSHA256Response, error) {
+func (a *Authenticator) GetPGSHA256Hash(_ context.Context, req *pb.PGSHA256HashRequest) (*pb.PGSHA256Response, error) {
 	a.counterMutex.Lock()
```

Comment:
Should the counter increment be wrapped in some method to make sure it's always done correctly?

Reply:
Sure, added in https://github.com/approzium/approzium/pull/72/commits/e7466a8d0a700c616244b49d085c86c2aff23616. :-)